### PR TITLE
KOGITO-3268 Trusty UI: Rely to enums for remote data status handling

### PR DESCRIPTION
**Pull request by kelvah**
JIRA ticket: https://issues.redhat.com/browse/KOGITO-3268

This PR replaces `SaliencyStatus` and `RemoteDataStatus` string literals with enums.

### CODE REVIEW

File: ui-packages/packages/trusty/src/types.ts [hunk 1]
```diff
@@ -78,8 +85,14 @@ export interface Saliency {
   outcomeId: string;
   featureImportance: FeatureScores[];
 }
+export enum SaliencyStatus {
```
Comment (manstis) [inline ui-packages/packages/trusty/src/types.ts:90]: Any reason you preferred to use an enumeration with strings for Salience Status yet numbers for Remote Data Status?

I am curious.

Comment (kelvah) [reply ui-packages/packages/trusty/src/types.ts:90]: Salience Status strings have to match with values retrieved from `/saliencies` endpoint. Remote Data Status instead is an _internal_ way to track various states of ajax calls.

**Review by manstis — APPROVED** [review]

**Comment by jstastny-cz** [comment]
I'm not able to review in timely manner, removing myself as a reviewer, please assess if you want to replace me with someone else (regardless their role in the team).

**Comment by sjamboro** [comment]
@kelvah feel free to add me as default reviewer. I should prepare test suite for trusty part of this project.

**Review by sjamboro — APPROVED** [review]

**Comment by kelvah** [comment]
Jenkins retest this please

**Comment by kelvah** [comment]
Jenkins retest this please

**Pull request merged**